plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1110,17 +1111,17 @@ blockly.block.item_consumed=If %1 is consumed
 blockly.block.item_damaged=If %3 is damaged, damage between %1 and %2
 blockly.block.item_in_inventory=If player has from %1 to %2 %3 in inventory
 blockly.block.tick=Each tick
-blockly.block.blockpos=Block position parameter named %1 with parameters %2 %3
+blockly.block.blockpos=Block position parameter named %1 %2 %3
```

Comment:
Can't we do without those `named` as well? (If not, proper verb is `called`).

Reply:
I would say so yep